hack/local-up-cluster.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright 2014 The Kubernetes Authors All rights reserved.
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
I changed this file - should I still keep the date to 2014? Should I only change the date for files that I've *added*? Thanks!